src/main/kotlin/com/atlassian/performance/tools/report/api/result/StatsMeter.kt

```diff
@@ -10,6 +11,45 @@ import java.time.Duration
 
 class StatsMeter {
 
+    private companion object {
+        private val DEFAULT_TRIMMING: OutlierTrimming = OutlierTrimming(0.0, 1.0)
+    }
+
+    internal fun measurePerformance(
+        result: EdibleResult,
+        centralTendencyMetric: UnivariateStatistic,
+        dispersionMetric: UnivariateStatistic,
+        actionToOutlier: Map<ActionType<*>, OutlierTrimming>
```

Comment:
I thought we are going for `trimmingPerType`. Can you search for all occurrences of `actionToOutlier` and rename them?